ceph.spec.in

```diff
@@ -218,7 +218,6 @@ BuildRequires:	systemd
 PreReq:		%fillup_prereq
 BuildRequires:	net-tools
 BuildRequires:	libbz2-devel
-BuildRequires:  btrfsprogs
```

Comment:
@tchaikov Need to also drop `BuildRequires: btrfs-progs` in the fedora/rhel conditional block about 20 lines down from here.